packages/calcite-components/src/components/input-time-zone/input-time-zone.tsx

```diff
@@ -77,6 +77,13 @@ export class InputTimeZone
   //
   //--------------------------------------------------------------------------
 
+  /**
```

Comment:
@geospatialem @DitwanP Would you mind reviewing this doc? It's based on `color-picker`'s [`clearable` prop doc](https://github.com/Esri/calcite-design-system/blob/main/packages/calcite-components/src/components/color-picker/color-picker.tsx#L130-L134), but I think both need some clarity tweaks.

Reply:
LGTM - think both use cases for `clearable` below and for `color-picker` although a bit wordy provide good context, but am flexible if @DitwanP has a recommended approach, too.